components/support/migration/build.gradle

```diff
@@ -89,5 +90,6 @@ dependencies {
 apply from: '../../../publish.gradle'
 ext.configurePublish(config.componentsGroupId, archivesBaseName, project.ext.description)
 
+// Do we need to remove this?
```

Comment:
I don't think we do - a-c still has the policy of generating these docs. So please just remove this new comment